x/evm/types/state_object.go

```diff
@@ -400,7 +400,16 @@ func (so *stateObject) GetCommittedState(_ ethstate.Database, key ethcmn.Hash) e
 func (so *stateObject) ReturnGas(gas *big.Int) {}
 
 func (so *stateObject) deepCopy(db *CommitStateDB) *stateObject {
-	newStateObj := newStateObject(db, so.account)
+	newAccount := ethermint.ProtoAccount().(*ethermint.EthAccount)
+	jsonAccount, err := so.account.MarshalJSON()
+	if err != nil {
+		return nil
+	}
+	err = newAccount.UnmarshalJSON(jsonAccount)
+	if err != nil {
+		return nil
+	}
```

Comment:
what is this for? why don't you copy the existing fields to a new `EthAccount`?

Reply:
The member variable PubKey of ETHAccount is of interface type, do you have a better suggestion for deep copy of interface type?

Reply:
@fedekunze Hi, what do you think of this solution? Can it be passed? Are there any possible problems with this solution?